app/templates/components/forms/wizard/basic-details-step.hbs

```diff
@@ -118,7 +118,7 @@
     <label>{{t 'Links & Social Media'}}</label>
     {{widgets/forms/link-input
       hasLinkName=true
-      fixedName=true
+      fixedName=false
```

Comment:
@harshitagupta30 i do not understand what you're telling. The code is fine. Checked the deployment link too. Works as expected. 

So, which exactly are you saying as not working ?